src/wui/game_summary.cc

```diff
@@ -190,18 +185,21 @@ void GameSummaryScreen::fill_data() {
 	// if not then the first line
 	uint32_t current_player_position = 0;
 
-	for (uintptr_t i = 0; i < players_status.size(); i++) {
-		Widelands::PlayerEndStatus pes = players_status.at(i);
-		if ((ipl != nullptr) && pes.player == ipl->player_number()) {
+	for (uintptr_t i = 0; i < game_.player_manager()->get_number_of_players(); ++i) {
+		const Widelands::PlayerEndStatus* pes = game_.player_manager()->get_player_end_status(i + 1);
```

Comment:
Sorry, I'm not sure about this, but it looks to me that if you do `i = 1 ; i <= …` here, then the correct index would be stored, so a lot of `index + 1`-s could be spared.